gitlab-pipeline/stage/init.yml

```diff
@@ -105,40 +116,23 @@ init:workspace:
     -   git fetch --tags origin
     - )
 
+    # Add Mender repositories.
+    - for repo in $(integration/extra/release_tool.py --list); do
+    -   checkout_repo \
+          git@github.com:mendersoftware/$repo \
+          go/src/github.com/mendersoftware/$repo \
+          $(eval echo \$$(repo_to_env_rev $repo)) ||
+        checkout_repo \
+          https://github.com/mendersoftware/$repo \
+          go/src/github.com/mendersoftware/$repo \
+          $(eval echo \$$(repo_to_env_rev $repo))
```

Comment:
I don't think the fallback is necessary. There is no reason not to clone it with the ssh syntax, it should work for all of them.

Reply:
I think you are right. Our convention was https for open source, ssh for private ones, but there is no good reason for that.

I'll amend, thanks!